src/features/stackQueue/components/StackQueueVisualizer.jsx

```
@@ -268,148 +301,210 @@ const StackQueueVisualizer = () => {
             setCurrentStep("Queue has elements, proceeding with dequeue");
           }
           break;
-        case 6: // Dequeue operation
+        case 6:
           setCurrentLine(22);
           setCurrentStep("Removing front element");
           const dequeuedValue = elements[0];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)